ios/Gekidou/Package.resolved

```diff
@@ -1,13 +1,94 @@
 {
```

Comment:
This is probably an unintended change. If so, can we restore this from whats on `master`?

Reply:
Sure